Firestore/Example/Tests/GoogleTest/GoogleTest.podspec

```diff
@@ -36,9 +36,14 @@ Google's C++ test framework.
   s.ios.deployment_target = '8.0'
   s.requires_arc = false
 
+  # Exclude include/gtest/internal/custom files from sources. These files cause
+  # problems because they have the same basenames as other headers (e.g.
+  # gtest.h). We don't need them because they're effectively empty: they're
+  # compile-time hooks for third-party customization that we don't use.
   s.source_files = [
     'googletest/src/*.{h,cc}',
-    'googletest/include/**/*.h',
+    'googletest/include/gtest/*.h',
+    'googletest/include/gtest/internal/*.h'
```

Comment:
Ditto flatten?  Lots of stuff in here looks like it should be just added as a single commit.

Also, I'm wondering whether this would be better - eventually - to land in the googletest repo and have those folks maintain the pod?

Reply:
I think that would be for the best, but there already exists a third-party GoogleTest pod (that's incompatible with iOS) that I don't want to mess with yet. This private spec allows us to declare a dependency similar to the way you can do `ExternalProject_Add` in CMake.